google/cloud/spanner/client_test.cc

```diff
@@ -981,6 +981,58 @@ TEST(ClientTest, CommitStats) {
   EXPECT_EQ(42, result->commit_stats->mutation_count);
 }
 
+TEST(ClientTest, MaxCommitDelayUnset) {
+  auto timestamp =
+      spanner_internal::TimestampFromRFC3339("2020-10-20T02:20:09.123Z");
+  ASSERT_STATUS_OK(timestamp);
+  CommitStats stats{42};
+
+  auto conn = std::make_shared<MockConnection>();
+  EXPECT_CALL(*conn, Commit)
+      .WillOnce([&timestamp, &stats](Connection::CommitParams const& cp) {
+        EXPECT_TRUE(cp.options.return_stats());
+        EXPECT_FALSE(
+            static_cast<Options>(cp.options).has<MaxCommitDelayOption>());
```

Comment:
We have only been making the "unset" expectations in one place ... `ClientTest.CommitAtLeastOnce`.  If you move this there, `MaxCommitDelayUnset` can go away.

Reply:
Done.